runtime-api/src/main/scala/io/coral/actors/database/CassandraActor.scala

```diff
@@ -30,7 +30,7 @@ object CassandraActor {
             seeds <- (json \ "seeds").extractOpt[List[String]]
             keyspace <- (json \ "keyspace").extractOpt[String]
         } yield {
-            (seeds, keyspace)
+            (seeds, (json \ "port").extractOpt[Int], keyspace)
```

Comment:
If it needs to be in the for section, the suggested line doesn't work because port is an optional parameter. It should then be wrapped in a Some. I thought the current solution is clearer and implemented it therefore this way. If you don't agree, please change the code because I won't be back until Monday and then the sprint has ended.